### DISCUSSION TITLE
Unable to use element-plus with vitepress: works in dev mode, but not after build

### DISCUSSION THREAD

**Discussion by akil-rails** [vuejs/vitepress#1379] [Q&A]
Refer https://github.com/vuejs/vitepress/issues/1164.

After upgrading to the latest versions of vitepress/vue, the built site fails to load the CSS, the build also returns a warning
```
✓ building client + server bundles...
⠋ rendering pages...[Vue warn]: Component <Anonymous> is missing template or render function.
[Vue warn]: Component <Anonymous> is missing template or render function.
[Vue warn]: Component <Anonymous> is missing template or render function.
[Vue warn]: Component <Anonymous> is missing template or render function.
✓ rendering pages...
```

Any idea on how to resolve this?

Reproducisble sample is at https://github.com/akil-rails/vitepress-with-element-plus

**Comment by MikeCZ23** [comment]
same. in localhost works, on github no
